app/jobs/govbox/check_messages_mapping_job.rb

```diff
@@ -0,0 +1,20 @@
+module Govbox
+  class CheckMessagesMappingJob < ApplicationJob
+    queue_as :default
+
+    def perform
+      unmapped_govbox_messages_sql =
+      "SELECT gm.id FROM govbox_messages gm JOIN govbox_folders gf ON gm.folder_id = gf.id
```

Comment:
Skus to pustit na stage/prod ze co to vrati a este by som tu dal ten limit na mesiac nech to nerobi pre celu db

Reply:
Staging aj prod 0 nenamapovanych sprav. @jsuchal